packages/lisk-transactions/src/0_transfer.js

```diff
@@ -12,13 +12,37 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+import cryptography from 'lisk-cryptography';
 import { BYTESIZES, TRANSFER_FEE } from './constants';
 import {
 	getAddressAndPublicKeyFromRecipientData,
 	wrapTransactionCreator,
+	validateAmount,
 } from './utils';
 
 const createAsset = data => {
+	if (data && data.length > 0) {
+		return { data };
+	}
+	return {};
+};
+
+const validateInputs = ({ amount, recipientId, recipientPublicKey, data }) => {
```

Comment:
This function should also validate the format of `recipientId` and `recipientPublicKey`.

Reply:
Addressed.